### PR TITLE
chore: updates to readme and funds

### PR DESCRIPTION
**Pull request by DavidMinarsch**

### CODE REVIEW

File: run_service.py [hunk 1]
```diff
@@ -57,10 +57,10 @@ def unit_to_wei(unit: float) -> int:
...
     return int(unit * 1e18)
 
-WALLET_TOPUP = unit_to_wei(0.05)
-MASTER_SAFE_TOPUP = unit_to_wei(0.01)
-SAFE_TOPUP = unit_to_wei(0.02)
-AGENT_TOPUP = unit_to_wei(0.01)
+WALLET_TOPUP = unit_to_wei(0.005)
+MASTER_SAFE_TOPUP = unit_to_wei(0.001)
+SAFE_TOPUP = unit_to_wei(0.002)
+AGENT_TOPUP = unit_to_wei(0.001)
```
Comment (DavidMinarsch) [inline run_service.py:63]: Is this feasible @dvilelaf ? .. I am just trying to get the funding requirements into a more reasonable area

File: README.md [hunk 2]
```diff
@@ -25,24 +31,15 @@ Ensure your machine satisfies the requirements:
...
-- Base
-- Celo
-
-See [here](https://github.com/dvilelaf/meme-ooorr?tab=readme-ov-file#user-flow) for what it does.
-
-Some thing you will need:
+## Setup Requirements
 
-- For the initial setup you will need to fund certain addresses with some funds when requested.
+- For the initial setup you will need to fund certain addresses with the following funds when requested: 0.05 ETH (Base chain) or x CELO (Celo chain). These quantities are based on the gas prices seen on the 1st half of Sept 2024 and may need to be revised. Additionally some quantity of OLAS bridged to Base or Celo if you want to stake.
```
Comment (DavidMinarsch) [inline README.md:36]: What are funding requirements for CELO?

File: run_service.py (hunk 1 above)
Comment (dvilelaf) [reply run_service.py:63]: I think so

File: README.md (hunk 2 above)
Comment (dvilelaf) [reply README.md:36]: HAvent tested on Celo. @0xArdi have you?

Comment (dvilelaf) [reply README.md:36]: I'll merge so I can tag 0.1.0, then we can fix on another PR

**Review by dvilelaf — APPROVED** [review]

**Pull request merged**